dev/com.ibm.ws.microprofile.config_fat/fat/src/com/ibm/ws/microprofile/config/fat/tests/ClassLoaderCacheTest.java

```diff
@@ -55,6 +58,10 @@
     public static final String WARB1_NAME = WARB1 + ".war";
     public static final String WARB2_NAME = WARB2 + ".war";
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests //selected combinations
+                    .with(new RepeatConfig14EE8("ClassLoaderCacheServer"));
```

Comment:
RepeatTests with only one config?

Reply:
yeah ... because doing so adds the id to the test result name; this is a config 1.1 bucket but only running against 1.4